airbyte-integrations/connectors/source-mongodb-v2/metadata.yaml

```diff
@@ -0,0 +1,20 @@
+data:
+  catalogs:
+    cloud:
+      dockerImageTag: 0.1.7
+      dockerRepository: airbyte/source-mongodb-strict-encrypt
```

Comment:
@alafanechere Also I think for strict encrypt we need to manually add a metadata.yml file for each.

These metadata files should
1. Have both cloud and oss catalogs disabled
2. Contain enough information to build the correct docker image
3. Have a comment acknowledging that the get deployed by their non strict encrypt sibling